rclcpp/test/test_node.cpp

```diff
@@ -25,13 +25,23 @@
 #include "rclcpp/scope_exit.hpp"
 #include "rclcpp/rclcpp.hpp"
 
+#include "rcpputils/filesystem_helper.hpp"
+
+
 class TestNode : public ::testing::Test
 {
 protected:
   static void SetUpTestCase()
   {
     rclcpp::init(0, nullptr);
   }
+
+  void SetUp() override
+  {
+    test_resources_path /= "test_node";
+  }
+
+  rcpputils::fs::path test_resources_path{TEST_RESOURCES_DIRECTORY};
```

Comment:
I hope [this](https://en.cppreference.com/w/cpp/experimental/fs) will enter in c++20 standard :smile:

Reply:
Hopefully, not sure when we'll get to switch though :(